autosklearn/pipeline/util.py

```diff
@@ -169,17 +169,25 @@ def _test_classifier_predict_proba(classifier, dataset='iris', sparse=False,
 
 
 def _test_preprocessing(Preprocessor, dataset='iris', make_sparse=False,
-                        train_size_maximum=150):
+                        train_size_maximum=150, task=None):
```

Comment:
Are all the changes in this file still necessary now that we have two distinct components for select rates?

Reply:
Not anymore, removed.